sdks/python/src/opik/llm_usage/opik_usage.py

```diff
@@ -75,10 +87,22 @@ def from_openai_completions_dict(cls, usage: Dict[str, Any]) -> "OpikUsage":
             usage
         )
 
+        # Extract audio tokens from token details
+        audio_input_tokens = None
+        audio_output_tokens = None
+        
+        if provider_usage.prompt_tokens_details and provider_usage.prompt_tokens_details.audio_tokens:
+            audio_input_tokens = provider_usage.prompt_tokens_details.audio_tokens
+            
+        if provider_usage.completion_tokens_details and provider_usage.completion_tokens_details.audio_tokens:
+            audio_output_tokens = provider_usage.completion_tokens_details.audio_tokens
+
         return cls(
             completion_tokens=provider_usage.completion_tokens,
             prompt_tokens=provider_usage.prompt_tokens,
             total_tokens=provider_usage.total_tokens,
+            audio_input_tokens=audio_input_tokens,
+            audio_output_tokens=audio_output_tokens,
             provider_usage=provider_usage,
```

Comment:
When converting to backend compatible dict, this provider_usage will contain keys like
`original_usage.completion_tokens_details.audio_tokens` - please use these values for price calculation on the BE side.